src/components/Upgrade/UpgradeOrganizationPanel.js

```diff
@@ -166,81 +168,46 @@ UpgradeOrganizationPanel.propTypes = {
   onClose: PropTypes.func.isRequired,
   repos: ReposListType,
   daoAddress: DaoAddressType.isRequired,
+  theme: PropTypes.object,
   wrapper: AragonType,
 }
 
-const AppVersion = ({
-  baseUrl,
-  contractAddress,
-  icons,
-  knownIcon,
-  name,
-  version,
-}) => {
-  const major = version.split('.')[0]
+const AppVersion = ({ repo }) => {
+  const { version } = repo
   return (
     <div
       css={`
-        display: flex;
+        display: inline-grid;
+        grid-template-columns: auto auto;
+        grid-gap: ${2 * GU}px;
         align-items: center;
-        margin: 10px 0;
+        margin: ${0.5 * GU}px 0;
       `}
     >
-      <div
-        css={`
-          width: 20px;
-          margin-right: 5px;
-        `}
-      >
-        v{major || version}
-      </div>
-      <ExternalLink
-        href={blockExplorerUrl('address', contractAddress, {
-          networkType: network.type,
-        })}
-        css={`
-          display: flex;
-          background: #daeaef;
-          border-radius: 3px;
-          align-items: center;
-          height: 22px;
-          align-items: center;
-          text-decoration: none;
-        `}
-      >
-        <AppIcon
-          app={{ baseUrl, icons }}
-          src={knownIcon || undefined}
-          radius={3}
-        />
-        <div
-          css={`
-            padding: 0 5px;
-          `}
-        >
-          {name}
-        </div>
-      </ExternalLink>
+      <div css="width: 26px;">{version}</div>
+      <RepoBadge repo={repo} />
     </div>
   )
 }
 
 AppVersion.propTypes = {
-  baseUrl: PropTypes.string.isRequired,
-  contractAddress: PropTypes.string.isRequired,
-  icons: PropTypes.array.isRequired,
-  knownIcon: PropTypes.string,
-  name: PropTypes.string.isRequired,
-  version: PropTypes.string.isRequired,
+  repo: PropTypes.shape({
+    content: RepoContentType.isRequired,
+    version: PropTypes.string.isRequired,
```

Comment:
`repoBaseUrl()` needs more than these fields (it takes the entire repo object)

Reply:
Are you sure? This is the way it was used before: https://github.com/aragon/aragon/pull/928/files#diff-e7868b75f1bc2f307a99861aa53cfa4cL52

Reply:
But [`getAppVersionData`](https://github.com/aragon/aragon/pull/928/files#diff-e7868b75f1bc2f307a99861aa53cfa4cL29) was returning more fields.

Reply:
Ahh, ok, I see, this bit was moved to the `RepoBadge` itself.